pkg/commands/process/settings/rules/ruby/rails/default_encryption.yml

```diff
@@ -32,4 +32,6 @@ metadata:
     Coming soon.
     -->
   dsr_id: DSR-10
+  cwe_id:
+    - 313
```

Comment:
Is this also CWE 311 "Missing Encryption of Sensitive Data"?

Unrelatedly, to confirm my understanding, are we saying this is CWE 313 ("Cleartext Storage in a File or on Disk") rather than CWE 312 ("Cleartext Storage of Sensitive Information") because this case is strictly a SQL database thing?

Reply:
I think 312 is more general here. It might not be a DB stored on disk. Also 312 is a child of 311 so I'd say that's implied without adding it explicitly.

Reply:
Ah, I missed the `stored_data_types` trigger on this (we don't document it) so I was avoiding any CWE about sensitive data. Will update to 312.